Mathlib/Tactic/TFAE.lean

```diff
@@ -203,7 +209,7 @@ def mkImplType (Pi : Q(Prop)) (arr : TSyntax ``impArrow) (Pj : Q(Prop)) : MetaM
 /-! # Tactic implementation -/
 
 elab_rules : tactic
-| `(tactic| tfae_have $[$h:ident : ]? $i:num $arr:impArrow $j:num) => do
+| `(tactic| tfae_have $[$h:ident : ]? $i:num $arr:impArrow $j:num $[:= $pf:term]?) => do
```

Comment:
It's there because currently we're consistent with "mathlib have" style syntax, where you can just do that. Did we/are we planning to banish that syntax for `have` as well?

Reply:
(Though if we're being consistent with `have`, maybe I should really push the `tfae_have 1 -> 2 := f ?a ?b` support as well...)

Reply:
So, to resolve this: yes, we will leave removing it to a future PR! :)